x-pack/test/functional/page_objects/upgrade_assistant_page.ts

```diff
@@ -29,6 +29,21 @@ export class UpgradeAssistantPageObject extends FtrService {
     });
   }
 
+  async navigateToFixDeprecationLogs() {
```

Comment:
To be aligned with the UI we could just call this `navigateToEsDeprecationLogs()` I think.